packages/manager/modules/pci/src/projects/project/private-registry/translations/Messages_fr_FR.json

```diff
@@ -44,7 +44,7 @@
   "private_registry_update_name_label": "Choisir un nom pour le Registre privé",
   "private_registry_update_success": "Le nom de Registre privé a été renommé en {{ newRegistryName }}",
   "private_registry_update_error": "Une erreur s'est produite lors du changement de nom du Registre privé {{ registryName }}: {{ message }}",
-  "private_registry_onboarding_success": "Votre registre privé {{ registryName }} a été créé. Veuillez générer des informations d'identification pour accéder à votre Registre",
+  "private_registry_onboarding_success": "Votre registre privé {{ registryName }} a été créé. Vous pouvez générer des informations d'identification pour accéder à votre Registre une fois qu'il est créé.",
```

Comment:
```suggestion
  "private_registry_onboarding_success": "Votre registre privé {{ registryName }} a été créé. Vous pouvez désormais générer des informations d'identification pour accéder à votre registre.",
```

Reply:
The meaning has to be "You can generate credentials to access your registry once it is ready." Since the button to "Create Credentials" appears only once the registry is created.

Reply:
Ok then I suggest
```js
Votre registre privé {{ registryName }} a été créé. Vous serez en mesure des informations d'identification pour accéder à votre registre une fois celui-ci disponible.
```